.example.env

```diff
@@ -0,0 +1,2 @@
+PORT=
+HOST=
```

Comment:
Let's keep a new line at the end of all files, as mentioned in the POSIX standard.